.changeset/sweet-fishes-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-pull-requests-board': minor
```

Comment:
This package is below version 1.0 so new features can be added with a `path` changeset.
```suggestion
'@backstage/plugin-github-pull-requests-board': patch
```